docs/hoon/twig/wut-test/bar-or.md

```diff
@@ -10,13 +10,25 @@ sort: 9
 ## Expands to
 
 ```
+:loop
 :ifno  p  
```

Comment:
To be more concrete, `|(a b c)` doesn't expand to `|-(?~(p | ?:(a & $(xx something)))`, but rather into `?:(a & ?:(b & ?:(c & |)))`. The `:loop` is purely compile-time.